plugins/cmd/chainlink-mercury/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"github.com/hashicorp/go-plugin"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/loop"
+)
+
+const (
+	loggerName = "PluginMercury"
+)
+
+func main() {
+	s := loop.MustNewStartedServer(loggerName)
```

Comment:
In this case, maybe it doesn't matter, but single letter variable names get really confusing. In cases like this we don't lose anything by using `server` instead of `s` and `plugin` instead of `p`.

Reply:
this is the style we have been using in other places for this kind of impl. want to be consistent.